.gitignore

```diff
@@ -0,0 +1,3 @@
+build
+TODO
```

Comment:
Will TODO be present/created in everyone's system who clone and run this repo?

Reply:
- Depends, I have it others might not but it can be good to have.